.build.octopus

```diff
@@ -0,0 +1,2 @@
+sha256:143934b13207a1f3a55f7c4ecfb4f8b991a4a92d186882aec63f8ce179da75bf
+octopus
```

Comment:
I am not sure how this happened but this file should not be checked in. Please remove it.

Reply:
Ah my bad, that change came in when I built the dev container. You can go ahead and approve this now